pkg/ddevapp/typo3.go

```diff
@@ -34,6 +34,11 @@ $GLOBALS['TYPO3_CONF_VARS']['MAIL']['transport_smtp_server'] = 'localhost:1025';
 
 $GLOBALS['TYPO3_CONF_VARS']['SYS']['devIPmask'] = '*';
 $GLOBALS['TYPO3_CONF_VARS']['SYS']['displayErrors'] = 1;
+
+// This GFX configuration allows processing by installed ImageMagick 6
+$GLOBALS['TYPO3_CONF_VARS']['GFX']['processor'] = 'im6';
+$GLOBALS['TYPO3_CONF_VARS']['GFX']['processor_path'] = '/usr/bin/';
+$GLOBALS['TYPO3_CONF_VARS']['GFX']['processor_path_lzw'] = '/usr/bin/';
```

Comment:
I see this already in the standard LocalConfiguration.php
``` 
   'GFX' => [
        'processor' => 'ImageMagick',
        'processor_allowTemporaryMasksAsPng' => false,
        'processor_colorspace' => 'sRGB',
        'processor_effects' => true,
        'processor_enabled' => true,
        'processor_path' => '/usr/bin/',
        'processor_path_lzw' => '/usr/bin/',
    ],
```

so the only change required is `$GLOBALS['TYPO3_CONF_VARS']['GFX']['processor'] = 'im6';`  right?

Is this change required for all TYPO3 projects? Or should the change be made in LocalConfiguration.php for a particular project?

Reply:
This is environment specific. As DDEV delivers ImageMagick6 at this location, this should be set.

Also some agencies might have an existing `LocalConfiguration.php` with setup for production which might differ, I would still set all three options, as they are environment specific, and therefore ddev specific.

Reply:
Sorry, I'm not understanding fully. The current configuration works to generate images, we had an issue about it many months ago. The difference is using im6 right?

Please note that if you're talking about something environment-specific, you can delete the `#ddev-generated` line from the AdditionalConfiguration and take ownership of the file and ddev won't touch it any more.

Reply:
The "environment specific" is ddev specific, as this file get's generated by ddev and therefore should configure the ddev environment.

The path and version are environment (ddev) specific.

If someone has configured `/usr/local/bin` within `LocalConfiguration.php` this will not work in current setup with ddev, as ddev has installed image magick 6 at `/usr/bin/`.

In my opinion ddev should configure TYPO3 through `AdditionalConfiguration.php` to work with ddev environment.

Reply:
OK, I'm good with overriding those values in AdditionalConfiguration.php, makes sense. 

I don't understand the switch from ImageMagick to im6 though. Is that something that all TYPO3 users would want?

Reply:
Yes.

Reply:
You could start a vote on Twitter or at typo3.slack.com :)

Reply:
I'd just ask in TYPO3 slack...